.github/workflows/snapshot-nhsn-data.yml

```diff
@@ -34,7 +34,9 @@ jobs:
         run: echo "FILE_NAME=nhsn-$(date +'%Y-%m-%d').csv" >> $GITHUB_ENV
 
       - name: Snapshot NHSN data
-        run: Rscript -e "nhsn_data <- RSocrata::read.socrata('https://data.cdc.gov/resource/mpgq-jmmr.csv'); 
+        run: Rscript -e "nhsn_data <- RSocrata::read.socrata('https://data.cdc.gov/resource/mpgq-jmmr.csv');
+                         nhsn_data <- nhsn_data[c("jurisdiction", "weekendingdate", "totalconfflunewadm", "totalconfc19newadm")];
+                         colnames(nhsn_data) <- c("Geographic aggregation", "Week Ending Date", "Total Influenza Admissions", "Total COVID-19 Admissions");
```

Comment:
keeping only the columns we need and renaming them to what you get when doing a manual download, using the data dictionary here: https://data.cdc.gov/Public-Health-Surveillance/Weekly-Hospital-Respiratory-Data-HRD-Metrics-by-Ju/mpgq-jmmr/about_data